docs/app/user-directory.md

```diff
@@ -2,54 +2,157 @@
 
-- **User Key** — The primary key of the user
-- **Last Page** — The last App page accessed by the user
-- **Last Access** — The timestamp of the user's last App or API action
+- **First Name** – The given name.
+- **Last Name** – The family/surname.
+- **Email** – A unique email address.
+- **Password** – A hashed system password.
+- **Avatar** – An image to represent the user.
+- **Location** – The city, country, office, or branch name.
+- **Title** – The professional staff title.
+- **Description** – A freeform text description.
+- **Tags** – Keywords for search-ability.
 
-::: tip System Fields
+### User Preferences
 
-Administrators have access to additional system fields.
+![User Preferences](https://cdn.directus.io/docs/v9/app-guide/user-directory/user-directory-20220222A/user-preferences-20220222A.webp)
+
+- **Language** – The preferred App language/locale.
+- **Theme** – Light or Dark mode (or based on system preferences).
+- **Multi-Factor Authentication** – Configuration for MFA.
+- **Email Notifications** – Receive emails for notifications.
+
+### Admin Options
+
+![Admin Options](https://cdn.directus.io/docs/v9/app-guide/user-directory/user-directory-20220222A/admin-options-20220222A.webp)
+
+- **Status** – Set User status Draft, Invited, Active, Suspended, Archived.
+- **Role** – Defines the User's Role.
+- **Token** – Accepts any string as user access token. At least 19-20 characters recommended, but we give you the tools
+  and flexibility to set this according to you own internal security policies.
+- **Provider** – _read-only:_ SSO provider associated with user. See our built-in [SSO options](/configuration/sso).
+- **External Identifier** – Displays external identifier generated by SSO provider.
+
+### Read-only Info
+
+![User Profile Sidebar Information](https://cdn.directus.io/docs/v9/app-guide/user-directory/user-directory-20220222A/user-profile-sidebar-information-20220222A.webp)
+
+Information in the Sidebar _(denoted by <span mi icon dark>info</span> when Sidebar is minimized)_ also includes the
+following read-only details:
+
+- **User Key** – The primary key of the user.
+- **Last Page** – The last App page accessed by the user.
+- **Last Access** – The timestamp of the user's last App or API action.
+
+## Extensibility Options
+
+Directus Core is completely open-source, modular and extensible. Extensions allow you to expand or modify any part of
+Directus to fit your needs. Here are some great resources to get started down that track.
+
+- [Extensions > Introduction](/extensions/introduction/)
+- [Extensions > Creating Extensions](/extensions/creating-extensions/)
+- [Contributing > Introduction](/contributing/introduction/)
+- [Contributing > Codebase Overview](/contributing/codebase-overview/)
+
+::: tip Accelerated Development
+
+Working on an enterprise project and looking to outsource or financially sponsor the development of a Shares extension?
```

Comment:
This feels a little odd in the context of the current page. Is this intended to be here, or is it a copy/paste error?

Reply:
I sliced it out. I would like to say that these are totally customizable and pick something out on every page that could be customized with an "Accelerated Development" plug, but I agree that perhaps is too forced as-is.